src/actinia_core/resources/common/redis_fluentd_logger_base.py

```diff
@@ -60,11 +58,54 @@ def __init__(self, config=None, user_id=None, fluent_sender=None):
         self.fluent_sender = fluent_sender
 
         # The fluentd sender
-        if fluent_sender is None and has_fluent:
-            self.fluent_sender = sender.FluentSender('actinia_logger', host=self.host, port=self.port)
+        if fluent_sender is None and global_config.LOG_INTERFACE == "fluentd":
+            self.fluent_sender = sender.FluentSender('actinia_logger',
+                                                     host=self.host,
+                                                     port=self.port)
 
-    def send_to_fluent(self, tag, data):
+    def _send_to_fluent(self, tag, data):
 
-        if has_fluent:
+        try:
             cur_time = int(time.time())
             self.fluent_sender.emit_with_time(tag, timestamp=cur_time, data=data)
+        except Exception as e:
+            log.error("%s is unable to connect to fluentd server host %s "
+                      + "port %i Error: %s, Content %s" %
+                      (tag, self.host, self.port, str(e), str(data)))
+
+        # keep this until sure that all logs are fetched if stdout log is set
+        # tags = ['RESOURCE_LOG', 'API_LOG', 'INFO', 'DEBUG']
+        if ('RESOURCE_LOG' not in tag and 'API_LOG' not in tag
+                and 'INFO' not in tag and 'DEBUG' not in tag):
+            print("WARNING: Some output might not be redirected to STDOUT:"
```

Comment:
Where would this be printed? Should it be logged? Again, I might be horribly stupid asking this.

Reply:
It is not stupid, I decided against use of the logger here because when it appears it marks a failure in logging itself. It should not appear at all but when it does, some logs were not collected properly which should then be fixed.

Reply:
Makes sense.